### PR TITLE
issue #185 Provide ansible tasks to regenerate CLI config and build via gradle

### PR DESCRIPTION
**Pull request by domdom82**
ansible role cli pulls the CLI image and runs it with a new default.props that is generated from a template. The template fills in the correct APIHOST for the environment we are deploying into. We then generate a new openwhisk tar.gz and copy it to nginx.

### CODE REVIEW

**Comment by domdom82** [comment]
WIP until gradle issues regarding Dockerfile location have been resolved.

**Comment by lehoanganh** [comment]
remove WIP, travis is running...

File: tools/cli/.project [hunk 1]
```diff
@@ -1,17 +0,0 @@
...
-		<buildCommand>
-			<name>org.python.pydev.PyDevBuilder</name>
-			<arguments>
-			</arguments>
-		</buildCommand>
-	</buildSpec>
-	<natures>
-		<nature>org.python.pydev.pythonNature</nature>
-	</natures>
-</projectDescription>
```
Comment (jeremiaswerner) [inline tools/cli/.project]: I think we should not delete this existing .project file. pls remove it from the commit.

.gitignore sounds good to me

**Comment by markusthoemmes** [comment]
rebased and merged in ac83db71824a5a51243d79cda21bfcceaceccff5

**Pull request closed without merging**